pages/establishment/update/views/index.jsx

```diff
@@ -105,7 +105,7 @@ class Repeat extends Component {
 
               <Fieldset
                 {...this.props}
-                model={mapKeys(authorisations.filter(auth => auth.type !== undefined), (val, key) => `authorisation-${authorisation.type}-${key}-${authorisation.id}`)}
+                model={mapKeys(authorisations.filter(auth => auth.type !== undefined)[index], (val, key) => `authorisation-${authorisation.type}-${key}-${authorisation.id}`)}
```

Comment:
Taking the same index after filtering the array is likely to break by returning an unexpected entry. I don't understand what the underlying issue is, but my best guess is that you want to retreive the entry at the index, and then cehck its type property.

Reply:
obsolete, it turned out to be another bug